packages/connectivity/src/scp-cf/destination/destination-from-env.ts

```diff
@@ -110,7 +110,7 @@ function validateDestinations(destinations: any[]) {
  */
 export function searchEnvVariablesForDestination(
   options: DestinationFetchOptions
-): Destination | undefined {
+): Destination | null {
```

Comment:
[q] why this change?

Reply:
Seems to violate our [styleguide](https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-undefined-for-explicit-absence-of-values) in my opinion. So if there is no good reason, this should be reverted.

Reply:
Thanks for the feedback! 
I changed this because all other three functions in `getDestination()` and also `getDestination` itself are returning `Destination | null` or promise of it. I think the style guide was more referring to the case for function parameters other than return type.
Also, correct me if I am wrong, I feel like the difference between `null` and `undefined` is that `null` means the value is explicitly empty, and `undefined` means the value has not been retrieved and assigned yet. [A post on StackOverflow](https://stackoverflow.com/a/5076962)

Reply:
I think the styleguide is not only meant for params, but we can have a discussion if it should be changed. I don't remember the arguments for the current way, but I remember there was a discussion back then 😊